components/Inventory/interfaces.rst

```diff
@@ -0,0 +1,88 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_inventory_model_inventory-unit-interface:
+
+InventoryUnitInterface
+~~~~~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single InventoryUnit.
+
+.. hint::
+    It also contains the default :doc:`/components/Inventory/state_machine`.
+
+.. note::
+    This interface extends :ref:`component_resource_model_timestampable-interface`. |br|
+    For more detailed information go to `Sylius API InventoryUnitInterface`_.
+
+.. _Sylius API InventoryUnitInterface: http://api.sylius.org/Sylius/Component/Inventory/Model/InventoryUnitInterface.html
+
+.. _component_inventory_model_stockable-interface:
+
+StockableInterface
+~~~~~~~~~~~~~~~~~~
+
+This interface provides basic operations for any model that can be stored.
+
+.. note::
+    For more detailed information go to `Sylius API StockableInterface`_.
+
+.. _Sylius API StockableInterface: http://api.sylius.org/Sylius/Component/Inventory/Model/StockableInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_inventory_checker_availability-checker-interface:
+
+AvailabilityCheckerInterface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This interface provides methods for checking availability of stockable objects.
+To characterize an object which is a **AvailabilityChecker** it needs to implement the ``AvailabilityCheckerInterface``
+
+.. note::
+    For more detailed information go to `Sylius API AvailabilityCheckerInterface`_.
+
+.. _Sylius API AvailabilityCheckerInterface: http://api.sylius.org/Sylius/Component/Inventory/Checker/AvailabilityCheckerInterface.html
+
+.. _component_inventory_factory_inventory-unit-factory-interface:
+
+InventoryUnitFactoryInterface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This interface is implemented by the service responsible for creating collection of new inventory units.
```

Comment:
try plural -> ` implemented by services responsible for`
and apply it for the ones below also.